applications/app/templates/serviceWorker.scala.js

```diff
@@ -150,6 +150,24 @@ var requestForAsset = (function () {
     }
 })();
 
+// handle requests for generic pages
+var requestForPage = function (event) {
```

Comment:
What's preventing this from being called on the dev blog as well? I might be missing something to do with `fetch` being a global promise that's resolved or something fun like that. Can there be a race condition?